audio_common_msgs/AudioData data
audio_common_msgs/AudioInfo audio_info
---
std_msgs/Bool success